lbrynet/tests/unit/components/test_Component_Manager.py

```diff
@@ -13,17 +14,32 @@ class TestComponentManager(unittest.TestCase):
     def setUp(self):
         mocks.mock_conf_settings(self)
         self.default_components_sort = [
-            [Components.DatabaseComponent,
-             Components.ExchangeRateManagerComponent,
-             Components.UPnPComponent],
-            [Components.DHTComponent,
-             Components.WalletComponent],
-            [Components.HashAnnouncerComponent],
-            [Components.SessionComponent],
-            [Components.PeerProtocolServerComponent,
-             Components.StreamIdentifierComponent],
-            [Components.FileManagerComponent],
-            [Components.ReflectorComponent]
+            [
+                Components.HeadersComponent,
+                Components.DatabaseComponent,
+                Components.ExchangeRateManagerComponent,
+                Components.PaymentRateComponent,
+                Components.RateLimiterComponent,
+                Components.UPnPComponent
+            ],
+            [
+                Components.DHTComponent,
+                Components.WalletComponent
+            ],
+            [
+                Components.BlobComponent,
+                Components.HashAnnouncerComponent
+            ],
+            [
+                Components.PeerProtocolServerComponent,
+                Components.StreamIdentifierComponent
+            ],
+            [
+                Components.FileManagerComponent
+            ],
+            [
+                Components.ReflectorComponent
+            ]
```

Comment:
I actually kinda like the original, it seems more compact, but don't feel super strongly about it.